File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/oracle/FloatTypes.kt
package dev.typr.foundations.docs.oracle

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class FloatTypes {
    //start
    val binaryFloat: OracleType<Float> = OracleTypes.binaryFloat
    val binaryDouble: OracleType<Double> = OracleTypes.binaryDouble
    val floatType: OracleType<Double> = OracleTypes.float_(126)  // FLOAT(126)
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/oracle/FloatTypes.scala
package dev.typr.foundationssc.docs.oracle
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object FloatTypes:
  // start
  val binaryFloat: OracleType[Float] = OracleTypes.binaryFloat
  val binaryDouble: OracleType[Double] = OracleTypes.binaryDouble
  val floatType: OracleType[Double] = OracleTypes.float_(126) // FLOAT(126)
  // stop
